sky/packages/sky/lib/src/gestures/lsq_solver.dart

```diff
@@ -0,0 +1,190 @@
+import "dart:math" as math;
+import "dart:typed_data";
+
+class Vector {
```

Comment:
Can we use https://www.dartdocs.org/documentation/vector_math/1.4.3/vector_math_64/Vector4-class.html instead?